music21/romanText/tsvConverter.py

```diff
@@ -166,81 +305,149 @@ def tabToM21(self):
         'vii'
         '''
 
-        if self.numeral:
-            if self.form:
-                if self.figbass:
-                    combined = ''.join([self.numeral, self.form, self.figbass])
-                else:
-                    combined = ''.join([self.numeral, self.form])
-            else:
-                combined = self.numeral
+        if self.numeral in ('@none', None):
+            thisEntry = harmony.NoChord()
```

Comment:
In DCML v1, there is a "length" column from which we can infer quarterLength.
In DCML v2, there is no such column, so inferring quarterLength becomes nontrivial. If we need it for some reason, that would require some work---unless you know a fast way of calculating it.
I will add a check so that we add quarterLength in case DCML is v1.

Reply:
I think that you're determining the quarter length for normal RNs just below this -- I'm pretty sure that code just needs to be unindented.  If NoChord doesn't have a length assigned to it, then it will always last 1 QuarterNote which can't be intended.

Reply:
Ah -- I see, you're getting the offsets properly but not the lengths.  What you might do then is in dcml v2 set all the quarterLengths to 0 -- that's better than showing them as quarter notes.  

Then when you're done with all parsing but before running makeMeasures, run Stream.extendDurations(harmony.Harmony, inPlace=True) if you'd like.  Or if measures are already made, it'll be a bit harder, but still possible.

Reply:
The v1 TabChord has a length attribute that is taken from the "length" column in the DCML v1 files. But in the v2 files there is no such column, so the v2 TabChord doesn't have any such attribute. But when we put them into music21 streams, they do have measureNumber and beat attributes. Is there any way of populating the quarterLength attribute of the items of a stream from their measureNumber and beat attributes?

Reply:
Currently, a column with lengths in quarters can be added to the TSVs optionally but I'm planning to make it a default in the upcoming weeks. If it helps the conversion, I can upgrade ABC v2 to that format?

Reply:
Yes, that would be great!